ffi/dart/tool/regenerate_hashes.dart

```diff
@@ -1,30 +1,48 @@
 import 'dart:io';
 
-import 'package:code_assets/code_assets.dart' show Architecture;
 import 'package:crypto/crypto.dart' show sha256;
 import 'package:icu4x/src/hook_helpers/version.dart' show version;
 
 Future<void> main(List<String> args) async {
   final httpClient = HttpClient();
 
   print('Checking hashes for $version');
-  final fileHashes = <(String, Architecture, String), String>{};
+  final fileHashes = <(String, String), String>{};
   final dynamicLibrary = File.fromUri(Directory.systemTemp.uri.resolve('lib'));
   await dynamicLibrary.create();
-  for (final os in ['linux', 'windows', 'fuchsia', 'android', 'macOS', 'iOS']) {
-    for (final architecture in Architecture.values) {
-      for (final libraryType in ['dynamic', 'static', 'static_data']) {
-        final target = [os, architecture, libraryType].join('_');
-        print('Checking hash for $target');
-        final success = await _fetchLibrary(target, httpClient, dynamicLibrary);
-        if (success) {
-          final bytes = await dynamicLibrary.readAsBytes();
-          final fileHash = sha256.convert(bytes).toString();
-          fileHashes[(os, architecture, libraryType)] = fileHash;
-          print('Hash is $fileHash');
-        } else {
-          print('Could not fetch library');
-        }
+  for (final rustTarget in [
+    'armv7-linux-androideabi',
+    'aarch64-linux-android',
+    'i686-linux-android',
+    'riscv64-linux-android',
+    'x86_64-linux-android',
+    'aarch64-unknown-fuchsia',
+    'x86_64-unknown-fuchsia',
+    'aarch64-apple-ios',
+    'x86_64-apple-ios',
+    'armv7-unknown-linux-gnueabihf',
+    'aarch64-unknown-linux-gnu',
+    'i686-unknown-linux-gnu',
+    'riscv32gc-unknown-linux-gnu',
+    'riscv64gc-unknown-linux-gnu',
+    'x86_64-unknown-linux-gnu',
+    'aarch64-apple-darwin',
+    'x86_64-apple-darwin',
+    'aarch64-pc-windows-msvc',
+    'i686-pc-windows-msvc',
+    'x86_64-pc-windows-msvc',
```

Comment:
Observation: You have this list, or things derived from this list, in multiple places. Seems fragile?

Reply:
the whole hash code needs to be cleaned up but I can only really work on it once we have published artifacts